lib/sourceParams.js

```diff
@@ -56,6 +56,18 @@ module.exports = (params, app, appSchema) => {
         params.mode = 'development'
       }
 
+      // handle webpack=verbose, --wp, -p cli flags
+      if (flags.webpack === 'verbose' || flags.wp || flags.p) {
+        params.logging.methods.verbose = true
+        params.logging.methods.verboseFile = false
+      }
+
+      // handle verbose-file, --wf, -f cli flags
+      if (flags.webpack === 'verbose-file' || flags.wf || flags.f) {
+        params.logging.methods.verbose = true
+        params.logging.methods.verboseFile = true
```

Comment:
the params you're modifying have to do with an unrelated feature (roosevelt's logger) and should not be modified in this way

a better way to go would be to modify `params.js.webpack` and add a new config called `verbose`. Set `verbose` to a value of either `true` or `'file'` to distinguish whether to log the verbose error or to do that plus write the file